src/components/avatar.tsx

```diff
@@ -0,0 +1,21 @@
+import { twJoin } from "tailwind-merge";
+import user from "@/assets/icons/user.svg";
```

Comment:
Suggestion: change it to a more meaningful name, for example: userDefaultImg.

Reply:
good suggestion